timescaledb/how-to-guides/multinode-timescaledb/multinode-grow-shrink.md

```diff
@@ -0,0 +1,28 @@
+# Grow and shrink multinode
+Quickly re-engineer cutting-edge e-markets after low-risk high-yield
```

Comment:
We have some things written on this topic here: https://docs.timescale.com/timescaledb/latest/how-to-guides/distributed-hypertables/manage-data-nodes/#moving-data-between-nodes

Should we move it here?